### ISSUE TITLE
Integration tests

### ISSUE THREAD

**Issue by kettanaito** [kettanaito/atomic-layout#15]
## What
Need to setup and write integration tests.

## Why
Atomic layout contributes to the end layout (UI) of the application. Its contribution needs to be tested to ensure it works as expected.

## How
1. Pick the technology (strongly suggest Cypress)
1. Configure the technology.
1. Write integration tests.

## Tests roadmap
- [x] Ensures "up" default behavior (x x x)
- [x] Ensures inclusive behavior (0 x 0)
- [x] Ensures bell behavior (x 0 x)
- [ ] Ensures responsive props (incl. all behaviors of props) (#23)
- [ ] Ensures composition (nested `<Composition/>`)
- [ ] Ensure custom breakpoint names
- [ ] Ensure custom measurement unit

**Comment by kettanaito** [comment]
I may take this setup as the part of #17.

**Comment by kettanaito** [comment]
I've installed and configured Cypress, and currently experience the following issue:
```
tests?p=cypress/integration/index.js-715:2609 It looks like there are several instances of 'styled-components' initialized in this application. This may cause dynamic styles not rendering properly, errors happening during rehydration process and makes you application bigger without a good reason.

See https://s-c.sh/2BAXzed for more info.
```

**Comment by kettanaito** [comment]
Multiple styled-components instances were caused by importing `Layout` class (from the built library) from within the Cypress commands directory. Layout imports `styled-components` under the hood, therefore Cypress ended up with multiple instances of styled-components.

The solution was to get the breakpoints objects directly from source of the library, to prevent importing extra instance of styled-components. Overall, that solved multiple instances problem, but tests still behaved wrong in Cypress.

Therefore, I've established a separate app which loads the built library and has test scenarios configured. Then Cypress visits that app and tests there to ensure proper styling, behavior and actually end-to-end usage.
